resources/templates/esp32/c_cpp_properties_linux.json

```diff
@@ -0,0 +1,22 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "defines": [
+                "ARDUINO=10800"
+            ],
+            "includePath": [
+                "${workspaceFolder}",
+                "{ROOTPATH}/.arduino15/packages/AZ3166/tools/**",

```

Comment:
Why we need to include "az3166" libraries in ESP32 project?

Reply:
It is a bug. Fixed.